docs/quickstart.md

```diff
@@ -28,6 +28,12 @@ or mix the two modes to override parameters in a config file.
 
 ### Train Pipeline
 
+Before we continue into specific instructions for training models please take into account
+that if you're working with data from WMT18 or later, you must make sure to pass the 
+`--wmt18-format True` flag. Either through yaml, or the CLI. This happens because WMT18 
```

Comment:
Just `--wmt18-format` should work.

Reply:
@captainvera , do you think it would make sense to change the config files so that all train using wmt18 data? That avoids the need to download to different datasets to run the examples